scripts/release.sh

```diff
@@ -36,7 +36,6 @@ function pub_pypi {
     # publish to pypi
     clean_build
     cp extra-requirements.txt jina/resources/
-    python setup.py sdist
```

Comment:
Actually I think this should not be removed, need to try out

Reply:
how can we try this out?